Task1/ClassLibrary/StartUp.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using Microsoft.Extensions.Configuration;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace LoggerClassLibrary
+{
+    class StartUp
+    {
+        public readonly Dictionary<LogLevel, List<string>> _logLevelToDestinationsDictionary = new Dictionary<LogLevel, List<string>>()
```

Comment:
You called `_logLevelToDestinationsDictionary` from the `_` which implies that it's a provate variable but define it with `public`. If you really need make it as `public`, use properties with public get and private set.